_generated/PublishSymbolsV2_Node20/PublishSymbols.ts

```diff
@@ -7,6 +7,12 @@ import * as clientToolRunner from "azure-pipelines-tasks-packaging-common/univer
 import * as tl from "azure-pipelines-task-lib/task";
 import { IExecSyncResult, IExecOptions } from "azure-pipelines-task-lib/toolrunner";
 
+const nodeVersion = parseInt(process.version.split('.')[0].replace('v', ''));
+if(nodeVersion < 16) {
+    tl.warning(tl.loc('NodeVersionSupport', nodeVersion));
+}
```

Comment:
Why is this code duplicated? Can't we have it in a common place and reuse that? The fact that we need to change this logic at 3 places is a code smell.

Reply:
Other two versions are in _generated folder and those are compiled files.